### PR TITLE
README: add rules for commiting to the repo

### PR DESCRIPTION
**Pull request by SvenRoederer**

### CODE REVIEW

**Comment by pmelange** [comment]
Addition is spelt wrong. Also, according to the openwrt rules, commits have to be signed. Are we going to use that rule too?

File: README.md [hunk 1]
```diff
@@ -24,3 +24,9 @@ See [LICENSE](LICENSE) file.
 ## Package Guidelines
 
 See [CONTRIBUTING.md](CONTRIBUTING.md) file.
+
+## If you have commit access (in addiotion to the rules of OpenWrt):
```
Comment (SvenRoederer) [inline README.md]: typo: addiotion

**Comment by SvenRoederer** [comment]
the "Patches and Pull requests:" I would just handle as noted in the referenced OpenWrt guideline (requiring and SoB)

**Comment by booo** [comment]
Are we talking about rules or are we talking about guidelines. I prefer the term guidelines here!

https://wikidiff.com/guideline/rule

In the end we use a distributed version control system and mistakes can be made.

Please link to content that explains your guidelines. The current three guidelines are not precise enough and contradict each other ("no force push" vs. "rebasing some branches").

There should be some guidelines about good commit messages incl. the body of a commit message.

**Comment by SvenRoederer** [comment]
>  The current three guidelines are not precise enough and contradict each other ("no force push" vs. "rebasing some branches").

Where is a contradiction? Rebasing an external branch of a PR onto one of our branches will not conflict with the "no-force" rule (see https://github.com/freifunk-berlin/firmware/pull/652#event-2163333583).

**Comment by pmelange** [comment]
I prefer the term guidelines too.  I also think we have a good start that we can clearly improve upon.  If nobody has an specific changes to the text, then I think we should merge this.

File: README.md (hunk 1 above)
Comment (SvenRoederer) [reply README.md]: typo fixed

**Comment by SvenRoederer** [comment]
changed wording to use the guideline term

**Comment by SvenRoederer** [comment]
merging, as there are no additional comments or complains

**Pull request merged**